src/formats/chr.h

```diff
@@ -40,7 +40,8 @@ typedef struct {
     sd_sprite *photo;                       ///< Pilot photo
     float winnings_multiplier;              ///< Tournament winnings multiplier
     sd_chr_enemy *enemies[MAX_CHR_ENEMIES]; ///< List of enemy states in current tournament
-    unsigned int cutscene;                  ///< cutscene id for end of tournament
+    char bk_name[14];                       ///< cutscene bk for end of tournament
+    int tournament_id;                      ///< misnomer? set to 4 for WORLD.BK cutscene quirks
```

Comment:
It does correspond to TOURNAMENT_NUMBER in the tournament compiler LST files.

Reply:
Ah, thanks. I did a search for "TOURNAMENT_ID" and came up blank :P
will remove the misnomer comment.